axelrod/interaction_utils.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-
+"""
+Functions to calculate results from interactions. Interactions are lists of the
+form:
+
+    [('C', 'D'), ('D', 'C'),...]
+
+This is used by both the Match class and the ResultSet class which analyse
+interactions.
+"""
+from .game import Game
+from axelrod import Actions
+
+C, D = Actions.C, Actions.D
+
+def get_scores(interactions, game=None):
+    """Returns the scores of a given set of interactions."""
+    if not game:
+        game = Game()
+    return [game.score(plays) for plays in interactions]
+
+
+def get_final_score(interactions, game=None):
```

Comment:
Minor: `compute_final_score` or similar instead of `get_final_score`? It's technically not simply fetching a score.